client/ayon_core/plugins/publish/collect_input_representations_to_versions.py

```diff
@@ -1,7 +1,18 @@
+import uuid
+
 import ayon_api
 import pyblish.api
 
 
+def is_valid_uuid(value) -> bool:
```

Comment:
We might use `convert_entity_id` from `ayon_api`.

Reply:
I like that! :) Implemented with https://github.com/ynput/ayon-core/pull/761/commits/a782ede959d3689e27d06e6cc991d5d31b1a2741